nginx/datadog_checks/nginx/nginx.py

```diff
@@ -99,6 +101,8 @@ def check(self, instance):
             # These are all the endpoints we have to call to get the same data as we did with the old API
             # since we can't get everything in one place anymore.
             for endpoint, nest in chain(iteritems(PLUS_API_ENDPOINTS), iteritems(PLUS_API_STREAM_ENDPOINTS)):
+                if int(plus_api_version) < 3 and endpoint == "stream/zone_sync":
```

Comment:
We should not hardcode the endpoint here, what do you think of a refactor like this? This will be more maintanable in case we need to add more endpoints in the future.

```python
for endpoint, nest in get_plus_api_endpoints(plus_api_version):
   # Same content

# Somewhere else in that file:
def get_plus_api_endpoints(plus_api_version):
    endpoints = [PLUS_API_ENDPOINTS, PLUS_API_STREAM_ENDPOINTS]
    if int(plus_api_version) >= 3:
        endpoints.append(PLUS_API_V3_STREAM_ENDPOINTS)
    return chain.from_iterable(endpoints)


# And at the top of the file, the constants:
PLUS_API_ENDPOINTS = {
    ... # Don't change it
}
PLUS_API_STREAM_ENDPOINTS = {
    "stream/server_zones": ["stream", "server_zones"],
    "stream/upstreams": ["stream", "upstreams"],
}
PLUS_API_V3_STREAM_ENDPOINTS = {
    "stream/zone_sync": ["stream", "zone_sync"]
}
```

Reply:
Yeah, that will work too.